mobile/library/kotlin/io/envoyproxy/envoymobile/EngineBuilderAdminUtil.kt

```diff
@@ -0,0 +1,21 @@
+package io.envoyproxy.envoymobile
+
+/**
+ * Utility to enable admin interface.
+ */
+object EngineBuilderAdminUtil {
```

Comment:
optional, what do you think of just removing these interfaces for kotlin? AFIK we planned on removing overall but are blocked for swift because no stats, but we could remove from Java/C++ now and avoid this extra class.

Reply:
that said, these classes working vs not for the compiled in options are pretty darned amazing.  Nicely done!

Reply:
I know Rafal cares about keeping admin functionality available with build flags, so I'd rather not rip that out while he's out 😅

Reply:
ah totally fine - I thought he only cared about it for iOS.